network/network.go

```diff
@@ -35,6 +35,8 @@ const (
 	// Having a value of 2 means that 2 specific net-exporters need to be down
 	// for one net-exporter to not be dialed, without exposing very high cardinality metrics.
 	numNeighbours = 2
+
+	appName = "net-exporter"
```

Comment:
Super nit:
Can you move this up into the same block as "namespace" (Line 23)

Reply:
This is exactly the feedback I'm looking for. Thanks!